### ISSUE TITLE
Engine requirement higher as necessary 

### ISSUE THREAD

**Issue by blynx** [kaisermann/svelte-i18n#206]
I just upgraded to the latest version 3.5.1 from 3.4.X and got an unsupported environment error. `Expected version: >= 17.5.0 vs. Got: v16.14.0` with pnpm.

```
 ERR_PNPM_UNSUPPORTED_ENGINE  Unsupported environment (bad pnpm and/or Node.js version)

This error happened while installing a direct dependency of /Volumes/NAVIO/repos/pm-monorepo/pm-static-app

Your Node version is incompatible with "registry.npmjs.org/svelte-i18n/3.5.1".

Expected version: >= 17.5.0
Got: v16.14.0
```

I cloned the repo and ran the tests on node `16.14.0` and all passed so a lower requirement should be no problem. I guess that somehow slipped through?

I think the engine requirements should ideally expect the lowest node version possible. Though practically I think it is useful to lean on the LTS versions of node: https://github.com/nodejs/release#release-schedule. Node 16 is not yet EOL.

EDIT: A sveltekit boilerplate via `(p)npm create` defined `engine-strict=true` in `.npmrc` - I don't rely on that but someone else might ^^

cheers
Steffen

**Comment by kaisermann** [comment]
Hey @blynx 👋  my bad, you're right! I've released v3.5.2 with requiring node >= 16 🙏

**Comment by blynx** [comment]
Awesome!
